frontend/src/pages/article/components/author-details/author-details.tsx

```diff
@@ -22,12 +23,13 @@ const AuthorDetails: FC<Properties> = ({
   publishedAt = '2023-09-04T12:53:07.144Z',
   readingTime = '7 min',
   genre = 'Fiction',
+  avatarUrl = null,
```

Comment:
Please remove default value assignment from `publishedAt` and `genre` and also remove optionality from props type (for genre I believe as of now type of prop should be `string | null`